.gitignore

```diff
@@ -1,4 +1,5 @@
 /target
 /Cargo.lock
+/cache/target
```

Comment:
@AlexRaiteri why did you need this?

Reply:
Cache was a folder that was deleted, it is now useless and can be deleted from .gitignore.

Reply:
Perfect, then I'm going to remove it.
```suggestion
```